src/Symfony/Contracts/HttpClient/ResponseInterface.php

```diff
@@ -97,15 +97,15 @@ public function cancel(): void;
      *  - response_headers (array) - an array modelled after the special $http_response_header variable
      *  - start_time (float) - the time when the request was sent or 0.0 when it's pending
      *  - url (string) - the last effective URL of the request
-     *  - user_data (mixed|null) - the value of the "user_data" request option, null if not set
```

Comment:
I think we should keep all `mixed|null` when null is documented with a special behavior

Reply:
Hm.. but `null` is included in `mixed`. I dont think `mixed|null` make sense. However, I would be very happy to see a more specific definition. Im sure this `mixed` will not include any `callable`.

Reply:
Actually yes, callable is a very valid type here - this is really `mixed`.
mixed contains null, but here we're not describing types for the engine, but types for humans, and documenting that null means something special is valuable I believe.

Reply:
Using just `mixed` is fine for me. The `null` case is documented in the description anyway.